NineChronicles.Headless.Tests/GraphTypes/StandaloneQueryTest.cs

```diff
@@ -555,6 +556,81 @@ public async Task GetTx()
             Assert.Equal(transaction.Actions.First().PlainValue.Inspection, plainValue);
         }
 
+        [Fact]
+        public async Task Products()
+        {
+            var userPrivateKey = new PrivateKey();
+            var service = MakeMineChroniclesNodeService(userPrivateKey);
+            StandaloneContextFx.NineChroniclesNodeService = service;
+            StandaloneContextFx.BlockChain = service.Swarm.BlockChain;
+            var blockChain = service.BlockChain;
+
+            const string query = @"query {
+                products {
+                    sellerAgentAddress
+                    sellerAvatarAddress
+                    price
+                    itemUsable {
+                        itemId
+                        itemType
+                        itemSubType
+                    }
+                    costume {
+                        itemId
+                        itemType
+                        itemSubType
+                    }
+                }
+            }";
+            await blockChain.MineBlock(new Address());
```

Comment:
Is `BlockChain<T>.MineBlock()` needed in this test

Reply:
yes, It is required for setting up the shop fixture.

Reply:
In my look, it seems using `Fixtures.ShopStateFX()` and it set up in *genesis*. Is there another point to set up the shop state like block action or other action?